src/docs/common/callout-view.adoc

```diff
@@ -0,0 +1,2 @@
+<3> Use https://micronaut-projects.github.io/micronaut-views/latest/api/io/micronaut/views/View.html[View] annotation to specify which template to use to render the response.
```

Comment:
Use a caret symbol to make the link open in a new window
```suggestion
<3> Use https://micronaut-projects.github.io/micronaut-views/latest/api/io/micronaut/views/View.html[View^] annotation to specify which template to use to render the response.
```

Reply:
I don't we we should open links in new windows. I think we should let the reader choose and use the default. Our audience is tech savy. If they want to open links in new windows or tabs they know how to do that. I don't think we should use `target=”_blank”` in our links